import numpy as np

robot_params = {}

robot_params['limo0'] ={'dt': 0.01,
                         'k_p': 10.,
                         'k_th': 2,
                         'buffer_size': 3000}

robot_params['limo1'] = {'dt': 0.01,
                         'k_p': 10.,
                         'k_th': 2,
                         'buffer_size': 3000}

robot_params['limo2'] = {'dt': 0.01,
                         'k_p': 10.,
                         'k_th': 2,
                         'buffer_size': 3000}